### PR TITLE
Makes baothas curse even worse 👍 

### PR DESCRIPTION
**Pull request by wow50000**
## About The Pull Request
Changes baothas curse rolls to be 10-30 seconds instead of 15-90

## Why It's Good For The Game
I want the coom curse to cause more cooming 👍 the curse doesnt feel bad enough to play as

### CODE REVIEW

File: code/datums/gods/patrons/curses.dm [hunk 1]
```diff
@@ -215,7 +215,7 @@
 	owner.sexcon.arousal += 1
 
 	if(owner.mob_timers["baotha_curse"])
-		if(world.time < owner.mob_timers["baotha_curse"] + rand(15,90)SECONDS)
+		if(world.time < owner.mob_timers["baotha_curse"] + rand(10,20,30,40,50,60)SECONDS)
```
Comment (Nakalon) [inline code/datums/gods/patrons/curses.dm:218]: im pretty sure rand only takes 2 numbers, does this even compile?

Comment (Nakalon) [reply code/datums/gods/patrons/curses.dm:218]: no it doesnt
image

you baboon

Comment (wow50000) [reply code/datums/gods/patrons/curses.dm:218]: what if I used pick instead ?
image

Comment (wow50000) [reply code/datums/gods/patrons/curses.dm]: image
nvm

**Comment by Gyrandola** [comment]
Next time you push changes without even compiling you'll get git banned

**Pull request closed without merging**